### PR TITLE
Completed initial version for CardPlatform

### PR DESCRIPTION
**Pull request by st0014**
---
Please check the following conditions for your PR.

- [x] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [x] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

File: custom-connectors/CardPlatform/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "tabIndex": 2
+          },
+          "description": "API Key from CardPlatform",
+          "displayName": "API Key",
+          "tooltip": "API Key is sent after registation is complete"
+        }
+      }
+    },
+    "iconBrandColor": "#006b23",
+    "publisher": "CardPlatform"
```
Comment (sriyen-msft) [inline custom-connectors/CardPlatform/apiProperties.json:20]: @st0014 Please add `stackOwner` property as well.

Documentation - [Adding metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)

File: custom-connectors/CardPlatform/readme.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+## CardPlatform Adaptive Card Connector
```
Comment (sriyen-msft) [inline custom-connectors/CardPlatform/readme.md:1]: @st0014 The image URL in the readme.md are broken, please fix them.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: custom-connectors/CardPlatform/readme.md (hunk 2 above)
Comment (st0014) [reply custom-connectors/CardPlatform/readme.md:1]: Updated the readme.md file with the correct path to the images

File: custom-connectors/CardPlatform/apiProperties.json (hunk 1 above)
Comment (st0014) [reply custom-connectors/CardPlatform/apiProperties.json]: Updated apiProperties with stackOwner

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**